### DISCUSSION TITLE
I can't manage my domain because of an error

### DISCUSSION THREAD

**Discussion by gy** [vercel/community#6623] [Account Issue]
### Question

I am getting this page:

Application error: a client-side exception has occurred (see the browser console for more information).

<img width="1393" alt="image" src="https://github.com/vercel/community/assets/54836/5c4db60c-852c-47bd-aad7-e589f4204e1b">

**Comment by amyegan** [comment]
Hey @gy. Do you see any errors in the browser console when you try to visit that page?

**Reply by the94air** [reply]
Hi. Can confirm a similar issue. Loading some js chunk has failed and it returns 404. Due to that the domains page is not accessible at all. Cleaning cache isn't useful.
```
https://vercel.com/_next/static/chunks/app/app-future/%5Blang%5D/(dashboard-v2)/%5BteamSlug%5D/(account)/~/account/domains/loading-e92e69e5a3c843e1.js?dpl=SOME_KIND_OF_A_CACHE_ID_HERE
```
